juntagrico/forms.py

```diff
@@ -38,6 +38,39 @@ def clean_password_repeat(self):
         return self.data['passwordRepeat']
 
 
+class MemberCancellationForm(Form):
+    message = CharField(label=_('Mitteilung'), widget=Textarea)
+    iban = CharField(label=_('IBAN'),
+                     help_text=_('Bitte hinterlege oder überprüfe deine IBAN, damit deine {} ausbezahlt werden können.').format(Config.vocabulary('share_pl')))
```

Comment:
Don't show the IBAN field if shares are disabled or if member has no shares.

Reply:
done